spec.html

```diff
@@ -18738,7 +18745,8 @@ <h2>Syntax</h2>
         `super` Arguments[?Yield, ?Await]
 
       ImportCall[Yield, Await] :
-        `import` `(` AssignmentExpression[+In, ?Yield, ?Await] `)`
+        `import` `(` AssignmentExpression[+In, ?Yield, ?Await] `,`? `)`
```

Comment:
ESMeta fails parsing `` `,`? ``. Do we consider an optional "token" valid grammar?

Reply:
[5.1.5.3 Optional Symbols](https://tc39.es/ecma262/#sec-optional-symbols) says that the `opt` subscript can appear after a terminal or nonterminal. However, I think you have to go back to ES 5.1 to find it applied to a terminal.